test/message/esm_display_syntax_error.out

```diff
@@ -2,6 +2,7 @@
 file:///*/test/message/esm_display_syntax_error.mjs:3
 await async () => 0;
 ^^^^^
+
 SyntaxError: Unexpected reserved word
-    at internal/modules/esm/translators.js:*:*
+    at Loader.<anonymous> (internal/modules/esm/translators.js:*:*)
```

Comment:
nit: is it worth making the loader method a named function, just to make this exception a bit clearer?

Reply:
fixed in 6f59606a71